### PR TITLE
sort ADOPTERS file

### LINKED ISSUE #3870: List ADOPTERS alphabetically?
Lists in prometheus/prometheus are sorted alphabetically. Would that make sense here as well?

### PR DESCRIPTION
**Pull request by paulfantom**
Fix #3870

**Release Note Template (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by brancz** [comment]
lgtm

File: ADOPTERS.md [hunk 1]
```diff
@@ -14,6 +14,33 @@ This is a meta user; please feel free to document specific OpenShift users!
 
 All OpenShift clusters use the Prometheus Operator to manage the cluster monitoring stack as well as user workload monitoring. This means the Prometheus Operator's users include all OpenShift customers.
 
+## Giant Swarm
```
Comment (lilic) [inline ADOPTERS.md:17]: All OpenShift Users might be a bit misleading if this sorted? :D Maybe we can rename All OpenShift users -> OpenShift users and sort it :)

Comment (QuentinBisson) [reply ADOPTERS.md:17]: I agree that putting the product name / company first makes sense here

Comment (simonpasquier) [reply ADOPTERS.md:17]: Agree too, what about s/All OpenShift Users/OpenShift/

Comment (paulfantom) [reply ADOPTERS.md:5]: Fixed :)

**Review by simonpasquier — APPROVED** [review]
lgtm

**Pull request merged**